controllers/suggestions/suggest_apiwrite_review_controller.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import random
+import string
+from datetime import datetime, timedelta
+from google.appengine.ext import ndb
+
+from consts.account_permissions import AccountPermissions
+from consts.auth_type import AuthType
+from controllers.suggestions.suggestions_review_base_controller import \
+    SuggestionsReviewBaseController
+from models.api_auth_access import ApiAuthAccess
+from models.event import Event
+from models.suggestion import Suggestion
+from template_engine import jinja2_engine
+
+
+class SuggestApiWriteReviewController(SuggestionsReviewBaseController):
+
+    def __init__(self, *args, **kw):
+        self.REQUIRED_PERMISSIONS.append(AccountPermissions.REVIEW_APIWRITE)
+        super(SuggestApiWriteReviewController, self).__init__(*args, **kw)
+
+    def get(self):
+        suggestions = Suggestion.query().filter(
+            Suggestion.review_state == Suggestion.REVIEW_PENDING).filter(
+            Suggestion.target_model == "api_auth_access").fetch()
+        suggestions = [self._ids_and_events(suggestion) for suggestion in suggestions]
+
+        self.template_values.update({
+            'success': self.request.get("success"),
+            'suggestions': suggestions,
+            'auth_names': AuthType.type_names,
+        })
+        self.response.out.write(
+            jinja2_engine.render('suggest_apiwrite_review_list.html', self.template_values))
+
+    def post(self):
+        self.verify_permissions()
+        suggestion = Suggestion.get_by_id(int(self.request.get("suggestion_id")))
+        verdict = self.request.get("verdict")
+        if verdict == "accept":
+            user = suggestion.author.get()
```

Comment:
All this should be in a transaction to ensure it's treated as an atomic operation. See https://github.com/the-blue-alliance/the-blue-alliance/blob/master/controllers/suggestions/suggest_team_media_review_controller.py#L147